media-sound/qmmp/qmmp-2.2.3.ebuild

```
@@ -0,0 +1,169 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+DESCRIPTION="Qt6-based audio player with winamp/xmms skins support"
+HOMEPAGE="https://qmmp.ylsoftware.com"
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="
+		https://qmmp.ylsoftware.com/files/${PN}/$(ver_cut 1-2)/${P}.tar.bz2
+		https://downloads.sourceforge.net/${PN}-dev/files/${P}.tar.bz2
+	"
+	KEYWORDS="~amd64 ~x86"
+else
+	inherit subversion
+	local BRANCH_VER="2.2" # NOTE: bump this to $(ver_cut 1-2) for live ebuild
+	ESVN_REPO_URI="svn://svn.code.sf.net/p/qmmp-dev/code/branches/${PN}-${BRANCH_VER}"
+fi
+
+LICENSE="CC-BY-SA-4.0 GPL-2+" # default skin & source code
+SLOT="0"
+# KEYWORDS further up
+IUSE="X aac +alsa analyzer archive bs2b cdda cddb cover crossfade cue curl +dbus
+doc enca ffmpeg flac game gnome jack ladspa libxmp lyrics +mad midi mms mpg123
+mplayer musepack notifier opus oss pipewire projectm pulseaudio qsui qtmedia
+scrobbler shout sid sndfile soxr stereo tray udisks +vorbis wavpack"
+
+REQUIRED_USE="
+	cddb? ( cdda )
+	gnome? ( dbus )
+	notifier? ( X )
+	shout? ( soxr vorbis )
+	udisks? ( dbus )
+"
+
+RDEPEND="
+	dev-qt/qtbase:6[X?,dbus?,gui,network,sqlite,widgets]
```

```suggestion
	dev-qt/qtbase:6[X?,dbus?,gui,network,sql,sqlite,widgets]
```
To help autounmask as sqlite requires sql